PBRVulkan/RayTracer/src/Assets/Shaders/Raytracer/Raytracing.rgen

```diff
@@ -42,31 +42,31 @@ void main()
 		float r1 = rnd(seed);
 		float r2 = rnd(seed);
 
-		vec2 jitter = vec2(r1, r2);
+		vec2 jitter     = vec2(r1, r2);
+		vec3 lensOffset = vec3(ubo.aperture / 2.0 * sampleUnitDisk(), 0.0);
 
 		vec2 pixel     = vec2(gl_LaunchIDNV.x, gl_LaunchIDNV.y) + jitter;
 		vec2 uv        = (pixel / gl_LaunchSizeNV.xy) * 2.0 - 1.0;
-		vec4 origin    = viewInv * vec4(0, 0, 0, 1);
-		vec4 target    = projInv * vec4(uv.x, uv.y, 1, 1);
-		vec4 direction = viewInv * vec4(normalize(target.xyz), 0);
-
-		// TODO Simulate a lens
+		vec4 origin    = viewInv * vec4(lensOffset, 1.0);
```

Comment:
I really thing you should divide commits. For example, this is actually a new feature despite of being small in code. I guess is working fine.